pelican/generators.py

```diff
@@ -675,8 +688,10 @@ def _process(arts):
         self.authors = list(self.authors.items())
         self.authors.sort()
 
-        self._update_context(('articles', 'dates', 'tags', 'categories',
-                              'authors', 'related_posts', 'drafts'))
+        self._update_context((
+            'articles', 'drafts', 'hidden_articles',
+            'dates', 'tags', 'categories',
+            'authors', 'related_posts'))
```

Comment:
This would also require an entry in the docs for [themes/common variables section](https://github.com/getpelican/pelican/blob/master/docs/themes.rst#common-variables)

Reply:
Completely unrelated but pretty sure the `related_posts` here is unused :). Probably, left over from ancient times where related plugin was in the core.